src/challenges.js

```diff
@@ -5,11 +5,13 @@
 const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
+  elements.forEach(cb);
```

Comment:
While it wasn't explicitly stated in the instructions, the hope was to do each of these without using array methods.